### ISSUE TITLE
Products option prices showing wrong currency symbol

### ISSUE THREAD

**Issue by QuantumEnt** [cubecart/v6#640]
![admin control panel-3 copy](https://cloud.githubusercontent.com/assets/13030487/8326242/a90c9958-1a57-11e5-9a3d-4563fb699397.jpg)
![order invoice - 150624-025410-3466](https://cloud.githubusercontent.com/assets/13030487/8326243/a912a884-1a57-11e5-8f7a-3bcf1d02c89a.jpg)

Product option prices when currency is set to £ are showing as $ on admin screen and on printed receipt.

**Comment by abrookbanks** [comment]
Please can you provide steps to reproduce this.

**Comment by QuantumEnt** [comment]
I have successfully replicated this issue as follow:
1. Set currency to US$
2. Purchase product - I have replicated this with the product 'Premium 'Ultra' Handwriting Font Service (Payment Only) - HWA5PU' selecting:

Make My Font An Example Font: Yes
Font Service Required: b. PLUS
PLUS Gift Certificate Purchase: No 

The order I just placed to replicate this can be seen in my order list as order number 150624-091512-1855
